app/celery/nightly_tasks.py

```diff
@@ -42,6 +44,19 @@ def _remove_csv_files(job_types):
         current_app.logger.info("Job ID {} has been removed from s3.".format(job.id))
 
 
+@notify_celery.task(name="cleanup-unfinished-jobs")
+def cleanup_unfinished_jobs():
+    now = datetime.utcnow()
+    jobs = dao_get_unfinished_jobs()
+    for job in jobs:
+        # The query already checks that the processing_finished time is null, so here we are saying
+        # if it started more than 4 hours ago, that's too long
+        acceptable_finish_time = job.processing_started + timedelta(minutes=5)
```

Comment:
This is more just a question on my part given I'm new to the code base:  what is the purpose of adding the 5 minute `timedelta` to the job processing start time?

Is it because the job itself is kicked off 5 minutes past midnight and accounting for that offset, or another reason?